### PR TITLE
Add/Update UWARC Social Information

### PR DESCRIPTION
**Pull request by zachomedia**
This pull request updates social information:

- Added a Twitter timeline to our homepage (note: if your screen is < 900px, the timeline will not be shown)
- Added information about our simplex frequency
- Reorganized and added information to our contact page.

### CODE REVIEW

**Comment by mtrberzi** [comment]
+1 nice work!

**Comment by ynasser** [comment]
Is the twitter feed supposed to be the top most thing? It pushes all the other content down below it. Shouldn't it be on the side? 

radio_page

**Comment by zachomedia** [comment]
@ynasser It should be off on the right-hand side. I've tested in both Chrome, Firefox on OS X and Ubuntu, as well as Safari on OS X  and it appears on the right.

Can I get you to try a couple of things?

1. Regenerate the Jekyll site
2. Clear you browser cache

Reload the page and let me know if it still appears above the content.

For reference, it should look like:

uwarc

File: index.html [hunk 1]
```diff
@@ -26,4 +34,6 @@
 			</li>
 		{% endfor %}
 	</ul>
+
+	<script>!function(d,s,id){var js,fjs=d.getElementsByTagName(s)[0],p=/^http:/.test(d.location)?'http':'https';if(!d.getElementById(id)){js=d.createElement(s);js.id=id;js.src=p+"://platform.twitter.com/widgets.js";fjs.parentNode.insertBefore(js,fjs);}}(document,"script","twitter-wjs");</script>
```
Comment (ehashman) [inline index.html]: lol this

**Comment by ehashman** [comment]
I can't test this on a CSC desktop, having issues installing Jekyll. Will take a look later.

Code looks good. Thanks for doing this, @zachomedia!

**Comment by prydom** [comment]
Compiled and tested, looks good. +1

(µBlock does block the twitter feed but that's setup dependent; I don't like like buttons.)

**Comment by ehashman** [comment]
Since @prydom is signing off on this, I'll merge and deploy.

**Pull request merged**